### ISSUE TITLE
Virtual Files no longer working on macOS

### ISSUE THREAD

**Issue by hosy** [owncloud/client#7659]
I am using virtual files on macOS for one year. Since client version 2.6.0 it seems like it is no longer working.
If I enable "Always available locally" it is working.

### Steps to reproduce
If I tap on o virtual `.owncloud` file, the file will not be replaced by the original file. I do not get an error message.
If I try to copy a virtual file, the file will not be copied.

![ezgif-7-2a1602f99a3f](https://user-images.githubusercontent.com/736109/71882967-b7820280-3135-11ea-93dc-afb72b99754e.gif)

### Client configuration
Client version:
2.6.0

Operating system:
macOS 10.15.1

OS language:
German

**Comment by HanaGemela** [comment]
I'm afraid that removing the copy of a virtual file is currently the expected behaviour https://github.com/owncloud/client/issues/4341#issuecomment-512949802

**Comment by michaelstingl** [comment]
> If I tap on o virtual `.owncloud` file, the file will not be replaced by the original file. I do not get an error message.

Copying isn't supported, but for non-copied files, this should work. Could you re-check, and provide logs in case this happens again.

**Comment by michaelstingl** [comment]
@TheOneRing You had an idea how to improve. Could you elaborate?

**Comment by hosy** [comment]
I have an update for this issue: I just tried to open a virtual ownCloud file, but nothing happened. I opened the ownCloud Desktop client settings and I saw, that my account needs a new authentication via OAuth in browser. After I authenticated I was able to open the virtual file. 

I think it would make sense to give the user an alert or feedback, when the user needs to reauthenticate again. Show an alert or directly open a OAuth browser site. But it is not so easy to determine, if nothing happens.
